test/statmodels.jl

```diff
@@ -82,6 +82,12 @@ ct = CoefTable(m, ["Estimate", "Stderror", "df", "p"], [], 4)
      df = 0.2422083248151139, p = 0.4530583319523316)
 ]
 
+ct = CoefTable(rand(1,100), ["c$i" for i in 1:100], [], 4)
+sct = sprint(show, ct, context=:limit=>false)
+@test length(first(split(sct, '\n'))) > 900
+sct = sprint(show, ct, context=:limit=>true)
+@test length(first(split(sct, '\n'))) < 200
```

Comment:
Can you check the exact output?

Reply:
I would not know what are the CI terminal dimensions.

Reply:
I think the default is returned by `displaysize`, i.e. `(24, 80)`.